### PR TITLE
Add item selector

### PR DESCRIPTION
**Pull request by hobbitronics**
- move ItemSelector into ClaimForm

### CODE REVIEW

File: src/components/forms/ClaimForm.svelte [hunk 1]
```diff
@@ -23,10 +23,11 @@ import type { ClaimFormData } from './claims/types'
 import { assertHas } from '../../validation/assertions'
 import { Button, Form, IconButton, MoneyInput, TextArea } from '@silintl/ui-components'
 import { createEventDispatcher, onMount } from 'svelte'
+import ItemSelector from './claims/_components/ItemSelector.svelte'
```
Comment (Schparky) [inline src/components/forms/ClaimForm.svelte:26]: sort imports

File: src/components/forms/ClaimForm.svelte [hunk 2]
```diff
@@ -167,8 +168,10 @@ function onInfoClick(event: Event) {
 </script>
 
 <div class="w-50">
-  {#if !hideItemName}
+  {#if items.length < 1}
```
Comment (Schparky) [inline src/components/forms/ClaimForm.svelte:171]: Seems this is more conventional, but my personal preference would be `if items.length > 0` and reverse the conditions. Go with your own preference, though.
```suggestion
  {#if !items?.length}
```

File: src/components/forms/ClaimForm.svelte
Comment (Schparky) [inline src/components/forms/ClaimForm.svelte:1]: I like that this change didn't affect other instances of the ClaimForm component.

**Review by Schparky — APPROVED** [review]
nice

File: src/components/forms/ClaimForm.svelte
Comment (hobbitronics) [reply src/components/forms/ClaimForm.svelte:1]: In the end I was surprised how little complexity it added.

**Pull request merged**